core/java/iesi-core/src/main/java/io/metadew/iesi/script/execution/instruction/data/text/TextSubstring.java

```diff
@@ -14,24 +15,43 @@
     private final Pattern INPUT_PARAMETER_PATTERN = Pattern.compile("(?<" + FIRST_OPERATOR + ">.+)," +
             "\\s*(?<" + SECOND_OPERATOR + ">-?\\d+)\\s*,\\s*(?<" + THIRD_OPERATOR + ">-?\\d+)\\s*");
 
+    private final Pattern INPUT_PARAMETER_PATTERN_TWO_ARGUMENTS = Pattern.compile("(?<" + FIRST_OPERATOR + ">.+)," +
+            "\\s*(?<" + SECOND_OPERATOR + ">-?\\d+)\\s*");
+
```

Comment:
Can these parameters be static?
Also, please rename 'INPUT_PARAMETER_PATTERN ' to 'THREE_ARGUMENT_PATTERN' and
'INPUT_PARAMETER_PATTERN_TWO_ARGUMENTS' to  'TWO_ARGUMENT_PATTERN'

Reply:
yes absolutely

Reply:
Finally, these variables can't be static because I used the variables just above ( FIRST_OPERATOR, …) in this contrast
![image](https://user-images.githubusercontent.com/73652675/98972946-69871980-2513-11eb-86bd-3fbb94856ded.png)